### DISCUSSION TITLE
Which communities could benefit from Alkemio?

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#400] [Ideas]
A community question: what innovation communities or collaborations do you know that could really benefit from Alkemio? Who should we connect in next?

Share your thoughts and ideas!

**Comment by techsmyth** [comment]
Please share your ideas! This is a new platform, and we are looking to understand where the biggest value add is so that we can use that input to shape the roadmap. 

All suggestions much appreciated!
